src/component/handlers/edit/editOnInput.js

```diff
@@ -88,8 +89,41 @@ function editOnInput(editor: DraftEditor): void {
   var domSelection = global.getSelection();
 
   var {anchorNode} = domSelection;
-  if (anchorNode.nodeType !== Node.TEXT_NODE) {
-    return;
+  const isNotTextNode =
```

Comment:
This line looks like it came from https://github.com/spicyj/draft-js/commit/3b91e36317b59c4eee6cf719826f743e07c2567b, which actually appears to set `draft_killswitch_allow_nontextnodes` to `false`, which changes the functionality and might not be fully tested yet